lib/exgboost/training/callback.ex

```diff
@@ -229,14 +215,14 @@ defmodule EXGBoost.Training.Callback do
           metrics: metrics,
           meta_vars: %{
             monitor_metrics: %{period: period, filter: filter}
-          }
+          },
+          status: :cont
         } = state
       ) do
     if period != 0 and rem(iteration, period) == 0 do
-      metrics = Map.filter(metrics, filter)
-      IO.puts("Iteration #{iteration}: #{inspect(metrics)}")
+      IO.puts("Iteration #{iteration}: #{inspect(Map.filter(metrics, filter))}")
```

Comment:
Let me know if you have any opinions about improving this logging. It's pretty bare-bones right now. Not necessarily for this PR though

Reply:
I think bare-bones is fine. And yeah, not for this PR.

However, an example in the docs that overrides this callback to, say, use `Logger` might be quite instructive.

(Also this was such I nit-picky change on my part, lol. The code before was completely fine.)